content/monitoring/deploy-prometheus.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Deploy Prometheus"
+date: 2018-10-14T20:33:02-04:00
+weight: 10
+draft: false
+---
+
+#### Download Prometheus
+
+```
+curl -o prometheus-values.yaml https://raw.githubusercontent.com/helm/charts/master/stable/prometheus/values.yaml
+```
+
+Search for **# storageClass: "-"**, uncomment and change the value to **"prometheus"**. You will do this twice, under both **`server`** & **`alertmanager`** manifests
```

Comment:
I'm wondering if we can come up with a better way to parameterize this file... maybe provide our own copy with pre-populated values when possible, to at least reduce the number of edits? We have a way to store/serve files that we are using in the onug branch.